src/commands/txCommands/lib.test.js

```diff
@@ -0,0 +1,128 @@
+const txlib = require('./lib');
+
+describe('trn utils', () => {
+	it('compares two objects, returning an object composed of new keys or existing keys with changed msgstr values', () => {
+		const mock = [
+			{
+				same: { msgstr: 'same' },
+				different: { msgstr: '123' },
+			},
+			{
+				same: { msgstr: 'same' },
+				different: { msgstr: 'abc' },
+				new: { msgstr: 'new' },
+			},
+		];
+
+		const expected = {
+			different: { msgstr: 'abc' },
+			new: { msgstr: 'new' },
+		};
```

Comment:
reeeeally trivial, but you can make the mock/expected more readable by defining constants and using ES6 shorthand

```js
const same =  { msgstr: 'same' };
const different =  { msgstr: 'different' };
const new =  { msgstr: 'new' };

const mock = [{ same, different }, { same, different, new }];
const expected = [{ different, new }];
```